packages/contracts-bedrock/src/L1/OptimismPortal2.sol

```diff
@@ -79,7 +79,7 @@ contract OptimismPortal2 is Initializable, ResourceMetering, ISemver {
     DisputeGameFactory public disputeGameFactory;
 
     /// @notice A mapping of withdrawal hashes to `ProvenWithdrawal` data.
-    mapping(bytes32 => ProvenWithdrawal) public provenWithdrawals;
+    mapping(bytes32 => mapping(address => ProvenWithdrawal)) public provenWithdrawals;
```

Comment:
This change will require an update to`getTimeToFinalize` in viem/opstack. only other usage of this in viem/opstack is in legacy code for portal < 3

Reply:
Will also require changes to the SDK because `provenWithdrawals` is used in some places.